e2e/cli-e2e/tests/collect.spec.ts

```diff
@@ -55,12 +58,7 @@ describe('CLI collect', () => {
   it('should print report summary to stdout', async () => {
     const { code, stdout, stderr } = await executeProcess({
       command: 'npx',
-      args: [
-        '../../dist/packages/cli',
-        'collect',
-        '--verbose',
-        '--persist.format=stdout',
-      ],
+      args: [cliPath, 'collect', '--verbose', '--persist.format=stdout'],
```

Comment:
You could use
https://github.com/flowup/quality-metrics-cli/blob/090e34d64c6ccb8ced01a615ea071c2fda83ea97/packages/utils/src/lib/execute-process.ts#L189

to generate the args in a more maintainable way

Reply:
This test was written by @matejchalk, I merely extracted one of the arguments to a variable. What do you think about this suggestion?

From my point of view, having arguments as an explicit array increases the test readability which is what you want in test code.

Reply:
I also don't see the point of `objectToCliArgs`, to be honest. This CLI notation is what the user sees, which makes it more fitting for an E2E test, the wrapper function just obfuscates it.